app/components/Header.js

```diff
@@ -34,6 +34,7 @@ import styles from '../../styles/components/Header.module.less';
 import logo from '../../images/Tidepool_Logo_Light x2.png';
 
 import { remote } from 'electron';
+import {selectClinic} from '../actions/sync';
```

Comment:
```suggestion
```
This import is not actually used.

Reply:
Ah yeah, my VSCode auto-import plugin gets overzealous sometimes. Thanks for catching it.